lib/secure-boot-lib.robot

```diff
@@ -4,6 +4,21 @@ Documentation       Collection of keywords related to UEFI Secure Boot
 Resource            ../keywords.robot
 
 
+*** Variables ***
+${GOOD_KEYS_URL}=           https://cloud.3mdeb.com/index.php/s/SjM5dQGji4XDAni/download/good_keys.img
+${GOOD_KEYS_NAME}=          good_keys.img
+${GOOD_KEYS_SHA256}=        13de737ba50c8d14a88aaf5314a938fb6826e18ba8f337470ee490b17dd6bea8
+${NOT_SIGNED_URL}=          https://cloud.3mdeb.com/index.php/s/zmJXxGG4piGB2Me/download/not_signed.img
+${NOT_SIGNED_NAME}=         not_signed.img
+${NOT_SIGNED_SHA256}=       15dc0a250b73c3132b1d7c5f8e81f00cc34d899c3ddecbb838a8cd0b66c4f608
+${BAD_KEYS_URL}=            https://cloud.3mdeb.com/index.php/s/BJPbSqRH6NdbRym/download/bad_keys.img
+${BAD_KEYS_NAME}=           bad_keys.img
+${BAD_KEYS_SHA256}=         6da92bd97d4b4ca645fa98dcdfdc0c6876959e5b815a36f1f7759bc5463e7b19
+${BAD_FORMAT_URL}=          https://cloud.3mdeb.com/index.php/s/AsBnATiHTZQ6jae/download/bad_format.img
+${BAD_FORMAT_NAME}=         bad_format.img
+${BAD_FORMAT_SHA256}=       59d17bc120dfd0f2e6948a2bfdbdf5fb06eddcb44f9a053a8e7b8f677e21858c
```

Comment:
Wouldn't it be better/easier to put all .cer files on one image/device? Is there any reason we don't do it this way?

Reply:
What is the idea for platforms without PiKVM?
If everything were on one image, it is possible to use a USB drive (some tests require the drive to be plugged in anyway).
Another option is to use the installed Linux to upload the necessary files to the USB.

Reply:
> Wouldn't it be better/easier to put all .cer files on one image/device? Is there any reason we don't do it this way?

That was the existing state of this MR, I have just reworked it. Feel free to improve that creating one image. You can ask @filipleple how he created these images. It could be easier in fact to have them on  a single disk.

I have assigned them labels with: ` sudo mlabel -i .<IMAGE>`

Reply:
> What is the idea for platforms without PiKVM?

No idea yet, single disk could be easier here indeed.

Reply:
I think we can leave it for later, we already have an issue for it.
https://github.com/Dasharo/open-source-firmware-validation/issues/68